homeassistant/components/gdacs/strings.json

```
@@ -3,9 +3,13 @@
     "step": {
       "user": {
         "title": "Fill in your filter details.",
-        "data": { "radius": "Radius" }
+        "data": {
+          "radius": "Radius"
+        }
       }
     },
-    "abort": { "already_configured": "Location is already configured." }
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
```

```suggestion
      "already_configured_service": "[%key:common::config_flow::abort::already_configured_service%]"
```
Shouldn't these match up.

---

No. Our helpers that abort have specific strings that we need to match, in this case `already_configured`.